command/results_xunit.go

```diff
@@ -167,19 +179,23 @@ func (c *xunitResults) parseAndUploadResults(ctx context.Context, conf *model.Ta
 		}
 	}
 
+	if len(tests) == 0 {
```

Comment:
I believe the go test parser doesn't error if there are no tests to upload right? They should be consistent